python-package/lets_plot/plot/plot.py

```diff
@@ -127,19 +143,26 @@ class GGBunch(FeatureSpec):
     Use show() to draw all plots in bunch.
 
     Examples
-    ---------
-    >>> import numpy as np
-    >>> import pandas as pd
-    >>> from lets_plot import *
-    >>> x = np.arange(100)
-    >>> y = np.random.normal(size=100)
-    >>> dat = pd.DataFrame({'x':x, 'y':y})
-    >>> g = ggplot(dat, mapping=aes('x', 'y')) + ggsize(150, 150)
-    >>> bunch = GGBunch()
-    >>> bunch.add_plot(g + geom_point(), 0, 0)
-    >>> bunch.add_plot(g + geom_histogram(bins=4), 150, 0)
-    >>> bunch.add_plot(g + geom_line(), 0, 150, 200, 100)
-    >>> bunch.show()
+    --------
+    .. jupyter-execute::
+        :linenos:
+        :emphasize-lines: 11 - 15
+
+        import numpy as np
+        from lets_plot import *
+        LetsPlot.setup_html()
+        np.random.seed(42)
+        n = 100
+        x = np.arange(n)
+        y = np.random.normal(size=n)
+        data = {'x':x, 'y':y}
+        w, h = 200, 150
+        g = ggplot(data, aes(x='x', y='y')) + ggsize(w, h)
+        bunch = GGBunch()
+        bunch.add_plot(g + geom_point(), 0, 0)
+        bunch.add_plot(g + geom_histogram(bins=3), w, 0)
```

Comment:
Maybe, `geom_density2d()` give us more interesting plot here? But if so we need also replace `x` by `np.linspace(-2, 2, n)` or something.